blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,176 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+==========
+Privileges
+==========
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
+
+When a user is created, he doesn't have any privileges. The superuser can then
+grant ``READ`` and ``WRITE`` privileges to this user.
+
+Please note that only the superuser can ``GRANT`` and ``REVOKE`` privileges.
+The superuser bypasses all privileges and can access all tables in the cluster.
+
+
+``GRANT``
+=========
+
+.. hide:
+
+    cr> CREATE table books;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE table school.courses;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE table school.teachers;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE table school.students;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE user wolfgang;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE user riley;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE user kala;
+    CREATE OK, 1 row affected (... sec)
+
+    cr> CREATE user will;
+    CREATE OK, 1 row affected (... sec)
+
+To grant a privilege to an existing user we use the :doc:`/sql/reference/grant` SQL statement,
+for example::
+
+    cr> GRANT READ ON TABLE books TO wolfgang;
+    GRANT OK, 1 row affected (... sec)
+
+Granting ``READ`` privilege on a table to a user, indicates that this user is allowed to run
+``SELECT`` statements on this table. Since privileges are only defined on a table level, this
+user will also be able to run other statements such as ``CREATE TABLE``,  ``CREATE FUNCTION``,
+``SHOW TABLES``, ``EXPLAIN`` etc.
+
+When a user creates a new table, he will have ``READ`` and ``WRITE`` privilege
+on that table.
+
+If a user with the username specified in the SQL statement does not exist the
+statement returns an error::
+
+    cr> GRANT WRITE ON TABLE books TO layla;
+    UserDoesNotExistException ...
+
+It is not possible to ``GRANT`` privileges to the built-in superuser ``crate``.
+
+If a table belongs to a specific schema, the ``READ`` privilege can be granted like this::
+
+    cr> GRANT READ ON TABLE school.courses TO wolfgang;
+    GRANT OK, 1 row affected (... sec)
+
+To grant ``WRITE`` privilege to user riley on the tables books, and school.courses we can
+use the following syntax::
+
+    cr> GRANT WRITE ON TABLE books, school.courses TO riley;
+    GRANT OK, 1 row affected (... sec)
+
+Granting ``WRITE`` privilege on a table to a user, indicates that this user is allowed to run
+``INSERT``, ``ALTER TABLE`` ``DELETE`` and ``DROP TABLE`` statements on this table.
```

Comment:
there are a lot more like  UPDATE, RESTORE, ... not sure if we should list all or rather say all but not READ/SELECT